src/interface/meta.thrift

```diff
@@ -64,21 +64,6 @@ enum AlterSchemaOp {
     UNKNOWN = 0x04,
 } (cpp.enum_strict)
 
-/**
-** GOD is A global senior administrator.like root of Linux systems.
-** ADMIN is an administrator for a given Graph Space.
-** USER is a normal user for a given Graph Space. A User can access (read and write) the data in the Graph Space.
-** GUEST is a read-only role for a given Graph Space. A Guest cannot modify the data in the Graph Space.
-** Refer to header file src/graph/PermissionManager.h for details.
-**/
-
-enum RoleType {
```

Comment:
why not just use those roles in common.thrift

Reply:
> why not just use those roles in common.thrift

Because using the RoleType in meta layer and graph layer.